editor/plugins/script_editor_plugin.cpp

```diff
@@ -4471,6 +4476,7 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 
 		disk_changed_list = memnew(Tree);
 		disk_changed_list->set_hide_root(true);
+		disk_changed_list->set_accessibility_name(TTR("The following files are newer on disk"));
```

Comment:
`NOTIFICATION_TRANSLATION_CHANGED` is automatically received when entering tree, so no need to set it here.